src/ziggurat/init.clj

```diff
@@ -177,13 +209,13 @@
   ([start-fn stop-fn stream-routes]
    (main start-fn stop-fn stream-routes []))
   ([start-fn stop-fn stream-routes actor-routes]
-   (main {:start-fn start-fn :stop-fn stop-fn :stream-routes stream-routes :actor-routes actor-routes}))
-  ([{:keys [start-fn stop-fn stream-routes actor-routes modes]}]
+   (main {:start-fn start-fn :stop-fn stop-fn :stream-routes stream-routes :actor-routes actor-routes :modes (all-modes-except-batch-consumer)}))
```

Comment:
Why do we need this?

Reply:
For backward compatibility as explained above.